microsoft/PowerToys
# [BugReportTool] Report if admin flags are set for executables

## Summary of the Pull Request

**What is this about:**
![image](https://user-images.githubusercontent.com/17161067/119004753-81929680-b997-11eb-871a-1baa6f7a7314.png)

**What is include in the PR:** 
- Report Compatibility tab info
- Rename `ReportRegistry` files to `RegistryUtils`
- Rename methods and files names to Pascal case

**How does someone test / validate:** 
- Create bug report
- Check the existence of the `admin-flags-info.txt` file in the report
- Verify if flags are set correctly

## Quality Checklist

- [X] **Linked issue:** #11256

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
